scripts/build-downstream-projects.sh

```diff
@@ -83,14 +83,16 @@ serum_dex() {
     rm -rf serum-dex
     git clone https://github.com/project-serum/serum-dex.git
     cd serum-dex
-    # TODO: Remove next line and debug build failure
-    git checkout 54400763dbcc64bc955621298f0bada33b591f53
```

Comment:
@CriesofCarrots fyi, this is no longer needed thanks to this downstream collaboration!: https://github.com/project-serum/serum-dex/issues/93

ref: https://github.com/solana-labs/solana/pull/14448

Reply:
Wonderful, thanks @ryoqun !